alembic/versions/9987c7c958c7_get_ci_error_messages.py

```diff
@@ -0,0 +1,71 @@
+"""Get CI error messages
+
+Revision ID: 9987c7c958c7
+Revises: 226fd58fe7e5
+Create Date: 2017-08-17 12:03:19.100006
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '9987c7c958c7'
+down_revision = '226fd58fe7e5'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+import fmn.lib
+import fmn.lib.models
+
+import fedmsg
+
+# Running this script actually produces fedmsg messages (since the db changes.)
+# Start fedmsg in active mode so that it talks to a fedmsg-relay
+try:
+    fedmsg.init(active=True)
+except ValueError:
+    pass
+
+path = 'fmn.rules:ci_step_complete'
+target = "Events on packages that I own"
+
+
+def upgrade():
+    engine = op.get_bind().engine
+    session = sa.orm.scoped_session(sa.orm.sessionmaker(bind=engine))
+
+    valid_paths = fmn.lib.load_rules(root='fmn.rules')
+
+    filters = session.query(fmn.lib.models.Filter).filter_by(name=target).all()
+    print("Found %i filters" % len(filters))
+
+    modified = 0
+    for filt in filters:
+        if len(filt.rules) < 15:
+            # Someone has changed this filter dramatically, so let's not mess
+            # with it.
+            print("Avoiding %r on %r.  Only %i rules present." % (
+                filt, filt.preference, len(filt.rules)))
+            continue
+
+        modified += 1
+        filt.add_rule(session, valid_paths, path, negated=True)
```

Comment:
To ignore the ones that have successfully completed, ie: notify about the ones that did not (that failed)